ReSharper.FSharp/test/data/features/quickFixes/removeXmlDoc/XmlDoc 02.fs.gold

```diff
@@ -0,0 +1,2 @@
+﻿() {caret}
```

Comment:
We should remove the space.

Please also consider this case in which keeping a whitespace line would be bad:

```fsharp
do
    /// 
    ()
```